pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.airlift.version>0.148</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.29</dep.slice.version>
-        <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.11.165</dep.aws-sdk.version>
```

Comment:
Presto has basic integration tests for S3 in `TestHiveClientS3`, though they are disabled by default since it requires an S3 account. The easiest way is to run that in your IDE, setting the parameters that are listed there. For setup, you'll need a minimal Hive installation with Hive metastore, with which you'll run the `create-test-s3.sql` Hive script to create the table pointing at S3.